src/components/Pages/ActionsPage/ActionsPage.js

```diff
@@ -292,7 +292,6 @@ class ActionsPage extends React.Component {
                 </center>
               </div>
               <HorizontalFilterBox
-                type="action"
```

Comment:
Why this line removed, was it causing problems?

Reply:
I was thinking of using the URL to automatically select  the appropriate button but Type prop will be less prone to bugs.  I will fix that.

Reply:
Please see latest commit